### ISSUE TITLE
[BUG] high pitched noise in silence

### ISSUE THREAD

**Issue by Zamundaaa** [ValveSoftware/SteamVR-for-Linux#316]
**Describe the bug**
When the sound on the Index is silent there's quite often a high pitched noise. It's not buzzing, seems to be one single frequency. That frequency also isn't always the same. Sometimes the noise starts and stops again several times a second (in regular intervals), sometimes it's continous.

**To Reproduce**
open the dashboard and click around a bit, between settings and the store for example. It will eventually start making the noise. This is not only happening in the dashboard but it seems like a reliable way to reproduce.

**Expected behavior**
In silence the loud speakers on the Index should stay, well, silent.

**System Information (please complete the following information):**
- Distribution: Manjaro
- SteamVR version: 1.30.32, the problem isn't really new though
- Steam client version: 1584730869
- Opted into Steam client beta?: Yes
- Graphics driver version:  Mesa 20.1.0-devel (git-500842399a) (LLVM 11.0.0)
- Gist for SteamVR System Information: https://gist.github.com/Zamundaaa/fe4988e8e750fe438a301396d84b6022

**Note:** Commenters who are also experiencing this issue are encouraged to include the "System Information" section in their replies.

**Comment by jug007** [comment]
Are you sure this is the Index itself?
The Lighthouses have a high pitch noise. Also there is the famous coil whining that a lot of graphics cards do nowadays.

**Comment by Zamundaaa** [comment]
yes, I am. It's far too loud to be coil whine from the GPU, my 5700 XT doesn't really have any and that would also be far too silent in comparison to the base stations. The sound I hear is in a far higher pitch than what the lighthouses make (they have more of a buzzing than a singular frequency anyways) and it comes directly from the loud speakers of the Index - if I angle the speakers up or take the headset off then it's gone.

I tested some more and have found out some more info:
- if I mute the sound _before_ the noise appears then I can't reproduce it
- if I mute the sound _while_ the noise is already there then it simply continues and is _not_ muted
-> it looks like it's always triggered by a sound playing and then not anymore (that's also why clicking around in the menu triggers it so often)
- it seems to randomly happen on either loud speaker, sometimes it's both, sometimes it's only one

**Comment by Goofybud16** [comment]
I've had this issue for a while as well in VRChat-- I find that moving the cursor over the quick menu buttons to make the little "boop" sounds will often make it go away.
